tutorials/klusterai-api/finetuning-sent-analysis.ipynb

```diff
@@ -308,36 +510,43 @@
     "id": "653ff10c-59bb-443d-b031-c6678744bdfc",
     "outputId": "391e1771-089c-43b2-851d-6c0c808e761f"
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "File uploaded successfully. File ID: 682388549bf57c3b70e5b99e\n"
+     ]
+    }
+   ],
    "source": [
-    "data_dir = 'finetuning/data/sentiment.jsonl'\n",
-    "\n",
+    "# Upload fine-tuning file\n",
     "with open(data_dir, 'rb') as file:\n",
-    "    upload_response = client_prod.files.create(\n",
+    "    upload_response = client.files.create(\n",
     "        file=file,\n",
     "        purpose=\"fine-tune\"\n",
     "    )\n",
+    "\n",
+    "    # Print job ID\n",
     "    file_id = upload_response.id\n",
-    "    print(f\"File uploaded successfully. File ID: {file_id}\")"
+    "    print(f\"File uploaded successfully. File ID: {file_id}\")\n"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "c9029a4f-8cfb-4193-b14b-7507006a969d",
-   "metadata": {
-    "id": "c9029a4f-8cfb-4193-b14b-7507006a969d"
-   },
+   "id": "9716bdd7-ba56-47ab-81e9-f12f2c29dbb8",
+   "metadata": {},
    "source": [
-    "Next, we'll submit the job to the kluster.ai fine-tuning API. Currently, two base models are supported for fine-tuning:\n",
-    "- klusterai/Meta-Llama-3.1-8B-Instruct-Turbo\n",
-    "- klusterai/Meta-Llama-3.3-70B-Instruct-Turbo\n",
+    "### Start the fine-tuning \n",
+    "\n",
+    "Once the fine-tuning file has been successfully uploaded, we're ready to start the fine-tuning job by providing the file ID we got in the previous step. To do so, we use the `fine_tuning.jobs.create` method, for which we also need to provide the model to fine-tune (check the <a href=\"/get-started/models/\" target=\"_blank\">Supported models</a> section). \n",
     "\n",
-    "If you specify a different base model, the fine-tuning job will fail. You can also tweak the hyperparameters (such as number of epochs, batch size, and learning rate) to adjust training time and potential performance gains. Remember that increasing the number of epochs will lead to longer training time but may result in higher performance. If you're unsure which hyperparameters to set, you can also comment them out to accept the default values."
+    "Optionally, you can also provide an object containing the <a href=\"/api-reference/reference/#create-a-fine-tuning-job\" target=\"_blank\">hyperparameters for fine-tuning</a>, such as number of epochs, batch size, and learning rate, to adjust training time and potential performance gains. Remember that increasing the number of epochs will lead to longer training time but may result in higher performance. If you're unsure which hyperparameters to set, you can also comment them to accept the default values."
```

Comment:
We should have this as a full link https://docs.kluster.ai/api-reference/reference/ for example, because it will not work from the Jupyter Notebook - gets 404.